src/main/kotlin/edu/kit/compiler/lex/Lexer.kt

```diff
@@ -2,6 +2,28 @@ package edu.kit.compiler.lex
 
 import edu.kit.compiler.Token
 
+private fun Char.isIdentifierStartChar() = when (this) {
```

Comment:
wollen wir hier auch inline nehmen? :D

Reply:
Mhh, theoretisch ist das in meinem kleinen Benchmark ~100-200ms besser, allerdings sagt IntelliJ dazu:
> Expected performance impact from inlining is insignificant. Inlining works best for functions with parameters of functional types

und ich glaube das ist praktisch komplett egal, weil du selten 100.000-Zeichen lange Identifier verwendest xP